make/configure.py

```diff
@@ -1238,6 +1238,10 @@ def createCLI( cross = None ):
     grp.add_argument( '--enable-x265', dest="enable_x265", default=True, action='store_true', help=(( 'enable %s' %h ) if h != argparse.SUPPRESS else h) )
     grp.add_argument( '--disable-x265', dest="enable_x265", action='store_false', help=(( 'disable %s' %h ) if h != argparse.SUPPRESS else h) )
 
+    h = IfHost( 'x265 NUMA support', '*-*-linux', none=argparse.SUPPRESS ).value
```

Comment:
If NUMA is Linux and Windows compatible:

`h = 'x265 NUMA support' if host.match( '*-*-linux*' ) or (cross is not None and 'mingw' in cross) else argparse.SUPPRESS`

If NUMA is Linux-only:

`h = 'x265 NUMA support' if host.match( '*-*-linux*' ) and cross is None else argparse.SUPPRESS`